### PR TITLE
Added .NET Core 3.1

### PR DESCRIPTION
**Pull request by alexx-grand**
Added .NET Core 3.1;
Null check of IScrollInfo.ScrollOwner

### CODE REVIEW

File: VirtualizingWrapPanel/VirtualizingWrapPanel/VirtualizingPanelBase.cs [hunk 1]
```diff
@@ -269,11 +269,11 @@ protected override Size MeasureOverride(Size availableSize)
             {
                 /* Sometimes the scrollbar gets hidden without any reason, to prevent
                  * a layout circle, return without any recalculation. */
-                if (ScrollOwner.ComputedVerticalScrollBarVisibility != Visibility.Visible && ViewportHeight < ExtentHeight)
+                if (ScrollOwner?.ComputedVerticalScrollBarVisibility != Visibility.Visible && ViewportHeight < ExtentHeight)
```
Comment (sbaeumlisberger) [inline VirtualizingWrapPanel/VirtualizingWrapPanel/VirtualizingPanelBase.cs:272]: This was already fixed in https://github.com/sbaeumlisberger/VirtualizingWrapPanel/commit/bcb5ac6a951ae4fd2dae4f5645d60e0a2fc8f02d

File: VirtualizingWrapPanel/VirtualizingWrapPanel/VirtualizingPanelBase.cs [hunk 2]
```diff
@@ -269,11 +269,11 @@ protected override Size MeasureOverride(Size availableSize)
...
                 {
                     return availableSize;
                 }
-                if (ScrollOwner.ComputedHorizontalScrollBarVisibility != Visibility.Visible && ViewportWidth < ExtentWidth)
+                if (ScrollOwner?.ComputedHorizontalScrollBarVisibility != Visibility.Visible && ViewportWidth < ExtentWidth)
```
Comment (sbaeumlisberger) [inline VirtualizingWrapPanel/VirtualizingWrapPanel/VirtualizingPanelBase.cs:276]: This was already fixed in https://github.com/sbaeumlisberger/VirtualizingWrapPanel/commit/bcb5ac6a951ae4fd2dae4f5645d60e0a2fc8f02d

File: VirtualizingWrapPanel/VirtualizingWrapPanel/VirtualizingWrapPanel.csproj [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.0;net45</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;netcoreapp3.0;net45</TargetFrameworks>
```
Comment (sbaeumlisberger) [inline VirtualizingWrapPanel/VirtualizingWrapPanel/VirtualizingWrapPanel.csproj:4]: What is the advantage of targeting netcoreapp3.0 and netcoreapp3.1?

**Comment by sbaeumlisberger** [comment]
Please open pull requests always to the develop branch. The master branch always contains the stable version released to nuget.

**Review by sbaeumlisberger — CHANGES_REQUESTED** [review]

File: VirtualizingWrapPanel/VirtualizingWrapPanel/VirtualizingWrapPanel.csproj (hunk 3 above)
Comment (alexx-grand) [reply VirtualizingWrapPanel/VirtualizingWrapPanel/VirtualizingWrapPanel.csproj:4]: Just for compatibility. I prefer to target to netcoreapp3.1.

**Comment by alexx-grand** [comment]
Ok. I'll create a new pull request to the develop branch.

**Pull request closed without merging**